documentation/commands/generate.md

```diff
@@ -0,0 +1,17 @@
+# exo generate
+
+_Generates docker compose and terraform files_
+
+Usage: `exo generate [command]`
+
+- `exo generate docker-compose` is automatically run before `exo run` and `exo test`
+- `exo deploy` will fail unless `exo generate terraform` is run beforehand
```

Comment:
I think these should go in a `Notes:` section at the bottom as these aren't about running this specific command but how it relates to other commands